ElGamalExt-Homomorphism.sln

```diff
@@ -16,17 +16,21 @@ Global
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{ACA46307-00C5-454F-A542-B9F179764945}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{ACA46307-00C5-454F-A542-B9F179764945}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{ACA46307-00C5-454F-A542-B9F179764945}.Deploy EGExt.Hom|Any CPU.ActiveCfg = Deploy EGExt.Hom|Any CPU
-		{ACA46307-00C5-454F-A542-B9F179764945}.Deploy EGExt.Hom|Any CPU.Build.0 = Deploy EGExt.Hom|Any CPU
+		{ACA46307-00C5-454F-A542-B9F179764945}.Deploy EGExt.Hom|Any CPU.ActiveCfg = Debug|Any CPU
+		{ACA46307-00C5-454F-A542-B9F179764945}.Deploy EGExt.Hom|Any CPU.Build.0 = Debug|Any CPU
```

Comment:
For the `Deploy EGExt.Hom` configuration to work in the new `.csprog`, certain changes to the `.csprog` needs to be made: https://stackoverflow.com/questions/43921992/how-can-i-use-beforebuild-and-afterbuild-targets-with-visual-studio-2017. These changes cause AppVeyor not to detect the `.csprog` automatically for patching. Therefore, if we want to keep this configuration, we will need to peform manual patching of the `.csprog` in the `appveyor.yml` file.